### ISSUE TITLE
Project API "merge_method"

### ISSUE THREAD

**Issue by mind-blowing** [gmessner/gitlab4j-api#174]
Can you please add possibility to read/update `merge_method` for the project.
https://docs.gitlab.com/ee/api/projects.html#edit-project

**Comment by gmessner** [comment]
Support for merge_method has been added to release 4.8.11.  Please verify and close when you have an opportunity.

**Comment by mind-blowing** [comment]
Unfortunately it doesn't works as expected, after fetching projects using `ProjectApi.getProjects()` and then using method `Project.getMergeMethod()` I see value `null`, and even if I will ignore `null` value and will try to update setting to proper value it will not works, after performing project update, value will be same as it was before.

Here is part of code that was used during verification:
```
val gitLabApi = new GitLabApi(gitLabProperties.getHost(), gitLabProperties.getPrivateToken());
val projectApi = gitLabApi.getProjectApi();

projectApi.getProjects().forEach(p -> {
    p.setIssuesEnabled(false);
    p.setOnlyAllowMergeIfAllDiscussionsAreResolved(true);
    p.setOnlyAllowMergeIfPipelineSucceeds(true);
    p.setPrintingMergeRequestLinkEnabled(true);
    p.setMergeMethod(Project.MergeMethod.FF);

    projectApi.updateProject(p);
});
```

I did verification with versions `4.8.11` and `4.8.12`, result is same, other settings updating without any problem, but `MergeMethod` is `null` during reading and it is not updating.

**Comment by gmessner** [comment]
@mind-blowing 
The "merge_method" attribute was not available until GitLab 10.7.0 (the latest release).  Looks like you figured this out and re-closed the issue.  I will add a line to the Javadocs on the create and update methods that highlight this.

**Comment by mind-blowing** [comment]
Yes, you are right, GitLab 10.7.0 was released few hours ago and my final verification was using 10.7.0, and after this I realized that it works fine and decided to close issue again.
